app/assets/javascripts/modules/Modules.OffenceSearchView.js

```diff
@@ -151,7 +151,7 @@ moj.Modules.OffenceSearchView = {
     this.$view.find('.fx-filters-display p').append(this.filterResults(options));
 
     this.$view.find('.fx-results-found p').empty();
-    this.$view.find('.fx-results-found p').append('Results found: ' + options.results.length + ' <a href="#noop" class="fx-clear-search">Clear search</a>');
+    this.$view.find('.fx-results-found p').append('Results found: ' + options.results.length + ' <a href="#noop" class="fx-clear-search hidden">Clear search</a>');
```

Comment:
Is this supposed to be hidden from the moment it's added?

Reply:
Yes, the default functionality is to hide the 'Clear search' link until there are search results.